packages/ui/src/components/va-toast/VaToast.vue

```diff
@@ -11,21 +11,21 @@
       ref="rootElement"
     >
       <div class="va-toast__group">
-        <h2 v-if="title" class="va-toast__title" v-text="title" />
```

Comment:
Why $props.[name] here and below? Your IDE swears on it cause of two defineComponent?

Reply:
For convenience :)